### DISCUSSION TITLE
Skip the landing page and go directly to docs

### DISCUSSION THREAD

**Discussion by nryberg** [google/docsy#1485] [Q&A] [answered]
While I love the landing splash page, I'd prefer that the site go directly to the documentation, do not pass go, do not collect $200.

I implemented this in a Docsy based site I worked on a couple of years ago, and now I can't remember how I forced the issue.  

Any suggestions?

**Comment by chalin** [comment]
Hi @nryberg. See:

- https://github.com/LisaFC/justdocs
- https://www.docsy.dev/docs/adding-content/content/#alternative-site-structure
- https://github.com/google/docsy/issues/173#issuecomment-579786866.

**Comment by honzik20** [comment, marked as answer]
Take a look at the second code block under https://www.docsy.dev/docs/adding-content/content/#alternative-site-structure. 

It's been implemented here: https://github.com/gwatts/mostlydocs/

**Comment by nryberg** [comment]
Thanks for the feedback!  Definitely shooting for simple rather than glitzy, and this is the right path forward.
